.github/workflows/pr-test.yml

```diff
@@ -85,3 +85,43 @@ jobs:
       - name: Run integration tests
         run: |
           pipenv run pytest integration_tests -k 'not api_key'
+
+  performance-tests:
+    strategy:
+      fail-fast: true
+      matrix:
+        python: ["3.7", "3.8", "3.9", "3.10"]
+    env:
+      working-directory: ./performance_tests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python }}
+      - uses: actions/setup-node@v2
+      - uses: azure/setup-helm@v1
+      - uses: dschep/install-pipenv-action@v1
+        with:
+          # version 2021.11.5, 2021.11.5.post0 do not work
+          version: 2021.5.29
+      - name: Build & install checkov package
+        run: |
+          pipenv --python ${{ matrix.python }}
+          pipenv run pip install --upgrade pip==21.1.1
+          pipenv run pip install pytest pytest-benchmark
+          pipenv run python setup.py sdist bdist_wheel
+          bash -c 'pipenv run pip install dist/checkov-*.whl'
```

Comment:
@tronxd I checked the tests out - we're not using the packaged checkov. So why do we need to build & install checkov?

Reply:
@nimrodkor The tests measure different checkov runners duration. so checkov is needed

Reply:
Checkov the code exists - why do we need the packaged version?
Are the tests bashing out and running `checkov ...`? I didn't see that